arm/Microsoft.ApiManagement/serviceResources/authorizationServers/readme.md

```diff
@@ -20,7 +20,7 @@ The following resources are required to be able to deploy this resource.
 | :-- | :-- | :-- | :-- | :-- |
 | `apiManagementServiceAuthorizationServerName` | string |  |  | Required. Identifier of the authorization server. |
 | `apiManagementServiceName` | string |  |  | Required. The name of the of the Api Management service. |
-| `authorizationEndpoint` | string |  |  | Required. OAuth authorization endpoint. See <http://tools.ietf.org/html/rfc6749#section-3.2>. |
+| `authorizationEndpoint` | string |  |  | Required. OAuth authorization endpoint. See http://tools.ietf.org/html/rfc6749#section-3.2. |
```

Comment:
Think this was done originally by the MD formatter. Its using the basic markdown syntax for links, as links should not be bare. <https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses>. 
However there is [auto generation of links in GitHub](Automatic linking for URLs
Any URL (like http://www.github.com/) will be automatically converted into a clickable link.), so maybe we turn off that rule in the MD formatter via settings.json?

Reply:
It does. I saw this in another place too. The autogernator does not add it

Reply:
done

Reply:
I didn't mean the readme generator, but on GitHubs side, they auto generate a http link to a clickable link (when they convert md to html). Using the md language native, its only supported with <link>. So I think its the update came from the MD autoformatter in VSCode (controlled by settings.json). So just thinking this can be omitted in settings.json. Ill make a separate PR for it.